src/microbe_stage/AgentProjectile.cs

```diff
@@ -35,12 +46,30 @@ public void OnBodyEntered(Node body)
             }
         }
 
+
+        DespawnTimer.Start();
```

Comment:
The particle emitter in the AgentProjectile should also be turned off here for the residual trailing effect. Also there's an extra whitespace on top of this call, should be removed.